### DISCUSSION TITLE
"Lit element transparently transmits a class style?"?

### DISCUSSION THREAD

**Discussion by whutBing** [lit/lit#3747] [Q&A]
I encapsulated a component An in lit-element. Component A has an attribute a-class to receive a style class name outside component A. how to make the external style act on only one shadow dom element of A (such as div). Note that it only works on one element.

**Comment by justinfagnani** [comment]
Have you looked at CSS Parts? https://developer.mozilla.org/en-US/docs/Web/CSS/::part

**Reply by augustjk** [reply]
There's also a great video on this topic: https://youtu.be/Xt7blcyuw5s
